Index: src/app/org/xmlcrm/app/remote/ChatService.java
===================================================================
--- src/app/org/xmlcrm/app/remote/ChatService.java	(revision 1067)
+++ src/app/org/xmlcrm/app/remote/ChatService.java	(working copy)
@@ -6,6 +6,8 @@
 import java.util.List;
 import java.util.LinkedHashMap;
 import java.util.LinkedList;
+import java.util.Calendar;
+import java.lang.Integer;
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
@@ -34,6 +36,27 @@
 	
 	private static LinkedHashMap<String,List<HashMap<String,Object>>> myChats = new LinkedHashMap<String,List<HashMap<String,Object>>>();
 	
+
+	private String parseDateAsTimeString() {
+		Calendar cal=Calendar.getInstance();
+		
+		int h = cal.get(Calendar.HOUR_OF_DAY);
+		int m = cal.get(Calendar.MINUTE);
+
+		String str_h,str_m;
+		if (h<10) 
+			str_h = "0"+Integer.toString(h);
+		else
+			str_h = Integer.toString(h);
+
+		if (m<10) 
+			str_m = "0"+Integer.toString(m);
+		else
+			str_m = Integer.toString(m);
+		
+		return str_h+':'+str_m;
+	}
+
 	/**
 	 * sends a Chat-Message
 	 * to all members of the Chatroom
@@ -58,6 +81,8 @@
 			//adding delimiter space, cause otherwise an emoticon in the last string would not be found
 			String messageText = messageMap.get(4).toString()+" ";
 			//log.error("messageText"+messageText);
+			//add server time
+			messageMap.put(1,parseDateAsTimeString());
 			LinkedList<String[]> parsedStringObjects = ChatString.getInstance().parseChatString(messageText);
 			//log.error("parsedStringObjects"+parsedStringObjects.size());
 			messageMap.put(9, parsedStringObjects);
@@ -241,6 +266,8 @@
 			//adding delimiter space, cause otherwise an emoticon in the last string would not be found
 			String messageText = messageMap.get(4).toString()+" ";
 			//log.error("messageText"+messageText);
+			//add server time
+			messageMap.put(1,parseDateAsTimeString());
 			LinkedList<String[]> parsedStringObjects = ChatString.getInstance().parseChatString(messageText);
 			//log.error("parsedStringObjects"+parsedStringObjects.size());
 			messageMap.put(9, parsedStringObjects);
